analysed-packages/libpng/version-1.6.39/libpng-1.6.39-SPDX2TV.spdx

```diff
@@ -0,0 +1,10918 @@
+Copyright (C) 2004 Cosmin Truta. </text>
+
+##File
+
+FileName: libpng-1.6.39.tar.xz/libpng-1.6.39/scripts/ltversion.m4
+SPDXID: SPDXRef-item494290
+FileChecksum: SHA1: 819eb60dafd5fd55525d62cca53bf10163202fa4
+FileChecksum: SHA256: 2a6baf40b64f34234e0b76c7ff62b01a1f6bbf60089f4f74506b49f800ffaf9e
+FileChecksum: MD5: 665ef728f05b03734804931333632de4
+LicenseConcluded: LicenseRef-FSFULLR-cf668dd067d1d7ab9e6adec72e2221c7
+LicenseInfoInFile: LicenseRef-FSFULLR
+LicenseInfoInFile: LicenseRef-fsf-unlimited
+FileCopyrightText: <text> Copyright (C) 2004, 2011-2019, 2021-2022 Free Software Foundation, Inc. Written by Scott James Remnant, 2004 </text>
+
+##File
+
+FileName: libpng-1.6.39.tar.xz/libpng-1.6.39/scripts/ltoptions.m4
+SPDXID: SPDXRef-item494242
+FileChecksum: SHA1: f98bd869d78cc476feee98f91ed334b315032c38
+FileChecksum: SHA256: f3d4ed0426501308028dd7690cf343cb280479d637a2ebee6c289ec2e708df3b
+FileChecksum: MD5: 98312682443b415dc42558ee6c7f25a9
+LicenseConcluded: LicenseRef-FSFULLR-cf668dd067d1d7ab9e6adec72e2221c7
+LicenseInfoInFile: LicenseRef-FSFULLR
+LicenseInfoInFile: LicenseRef-fsf-unlimited
+FileCopyrightText: <text> Copyright (C) 2004-2005, 2007-2009, 2011-2019, 2021-2022 Free Software Foundation, Inc. Written by Gary V. Vaughan, 2004 </text>
+
+##File
+
+FileName: libpng-1.6.39.tar.xz/libpng-1.6.39/scripts/ltsugar.m4
+SPDXID: SPDXRef-item494288
+FileChecksum: SHA1: 530ed09615ee6c7127c0c415e9a0356202dc443e
+FileChecksum: SHA256: 0896f153a5a40546566028a4272642ae291532f3e65c25fcae950c8812b8c265
+FileChecksum: MD5: 057a003c14ef81b6c16d1dc54d491105
+LicenseConcluded: LicenseRef-FSFULLR-cf668dd067d1d7ab9e6adec72e2221c7
+LicenseInfoInFile: LicenseRef-FSFULLR
+LicenseInfoInFile: LicenseRef-fsf-unlimited
+FileCopyrightText: <text> Copyright (C) 2004-2005, 2007-2008, 2011-2019, 2021-2022 Free Software Foundation, Inc. Written by Gary V. Vaughan, 2004 </text>
+
+##File
+
+FileName: libpng-1.6.39.tar.xz/libpng-1.6.39/scripts/lt~obsolete.m4
+SPDXID: SPDXRef-item494275
+FileChecksum: SHA1: 421180f15285f3375d6e716bff269af9b8df5c21
+FileChecksum: SHA256: 8533006830e1ea9625fc5e4c060e653eedf9d5464a9b2f5f494244ee272e2e2f
+FileChecksum: MD5: 42fc54566fd6a8055fad65f78f32960c
+LicenseConcluded: LicenseRef-FSFULLR-cf668dd067d1d7ab9e6adec72e2221c7
+LicenseInfoInFile: LicenseRef-FSFULLR
+LicenseInfoInFile: LicenseRef-fsf-unlimited
+FileCopyrightText: <text> Copyright (C) 2004-2005, 2007, 2009, 2011-2019, 2021-2022 Free Software Foundation, Inc. Written by Scott James Remnant, 2004. </text>
+
+##File
+
+FileName: libpng-1.6.39.tar.xz/libpng-1.6.39/configure
+SPDXID: SPDXRef-item494221
+FileChecksum: SHA1: 90118afeae01264a64c8edb682e0fc8533b579b6
+FileChecksum: SHA256: 3c2128bc797a54787ed243298911c002aa92cad778f9c2b3cbd134516fdcfbde
+FileChecksum: MD5: fa4bc28348325c42c656b41d8b5d4642
+LicenseConcluded: LicenseRef-FSFUL-924d9fc673a4618296ede854db59cec8 AND LicenseRef-GPL-2.0-or-later-WITH-Libtool-exception
```

Comment:
I think there is also the following statement in the file:
This config.status script is free software; the Free Software Foundation
gives unlimited permission to copy, distribute and modify it.
This shall be reflected to

Reply:
Thank you so much!